data/card_catalogue/food.json

```diff
@@ -0,0 +1,28 @@
+[
+  {
+    "title": "Spice Rack",
+    "upgrade": "Ambrosia",
+    "tier": "blue",
+    "details": {
+      "rules": {
+        "active": "Swap the flavours of two objects within your reach.",
+        "passive": ""
+      },
+      "flavour": "Mmm, Crayons",
+      "image": "assets/core/card/illustration/Spice Rack.png"
+    }
+  },
+  {
+    "title": "Ambrosia",
+    "upgrade": "",
+    "tier": "green",
+    "details": {
+      "rules": {
+        "active": "Summon a brick of Ambrosia. Any NPC who has tasted or smelt it before will find it irresistible.",
+        "passive": ""
+      },
+      "flavour": "Lifelong Ambrosia addicts scuttle around gourmet kitchens like rats, hoping to find tiny scraps.",
+      "image": "assets/core/card/illustration/Ambrosia.png"
+    }
+  }
+]
```

Comment:
moved these cards out of `npc/cook` to their own deck since they're not campaign-specific